Database/Table/MySQLTable.php

```diff
@@ -81,7 +81,10 @@ protected function getColumnForDdl(Column $column, array &$bindings): string
         $string .= $column->getUnsignedText();
         $string .= $column->isNullable() ? ' null' : ' not null';
         $string .= $this->getDefaultAsBindingOrText($column, $bindings);
-        if (isset($this->primaryKeyName) && $this->primaryKeyName == $column->getName()) {
+        if (
+            $this->primaryKeyAutoIncrement && isset($this->primaryKeyName) 
```

Comment:
```suggestion
            $this->primaryKeyAutoIncrement 
```

Looking at this, since it defaults to null, we don't actually need the call to isset, anymore. My bad code, your commit to fix it!